.github/workflows/pull_request.yml

```diff
@@ -12,9 +12,30 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+# A workflow run is made up of one or more jobs that can run sequentially
 jobs:
-  # This workflow contains a single job called "build"
+  ci_setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out forklift repository
+        uses: actions/checkout@v3
+
+      - name: Build and setup everything with baze
+        uses: kubev2v/forkliftci@main
```

Comment:
I have created a custom action for the setup of forklift ci https://github.com/kubev2v/forkliftci/pull/6/.